packages/valory/skills/decision_maker_abci/behaviours/sampling.py

```diff
@@ -43,7 +43,7 @@ def available_bets(self) -> Iterator[Bet]:
 
         # Note: the openingTimestamp is misleading as it is the closing timestamp of the bet
         if self.params.using_kelly:
-            # get only bets that close in the next 48 hours
+            # get only bets that close in the next `sample_bets_closing_days`
```

Comment:
The `sample_bets_closing_days` override can be set to a large value to test kelly for Polymarket.